src/App.tsx

```diff
@@ -1,8 +1,31 @@
 import './App.scss';
+import { Route, Routes } from 'react-router-dom';
+import { PeoplePage } from './components/PeoplePage';
+import { HomePage } from './components/HomePage';
+import { Header } from './components/UIComponents/Header';
+import Footer from './components/UIComponents/Footer';
+import NotFoundPage from './components/NotFoundPage';
 
 const App = () => (
   <div className="App">
-    <h1>People table</h1>
+    <Header />
+    <main>
+      <Routes>
+        <Route
+          path="/people"
+          element={<PeoplePage />}
+        />
+        <Route
+          path="/"
+          element={<HomePage />}
+        />
+        <Route
+          path="*"
+          element={<NotFoundPage />}
+        />
+      </Routes>
```

Comment:
Use [Navigate](https://reactrouter.com/docs/en/v6/components/navigate)) component to redirect from /home to /;